matplotcheck/base.py

```diff
@@ -468,10 +394,10 @@ def assert_legend_no_overlay_content(
         ---------
         m: string error message if assertion is not met
         """
-        plot_extent = self.ax.get_window_extent().get_points()
+        plot_extent = self.ax.get_window_extent(RendererBase()).get_points()
```

Comment:
can you please clarify what RendererBase() does / why it was was added @kysolvik ?

Reply:
@kysolvik can you please add a comment to the code where you call renderBase so we remember why it's there??